_episodes/16-motivation.md

```diff
@@ -110,6 +110,28 @@ that we *shouldn't* identify and reward what we value---but
 it's useful to check lessons against these points
 to make sure they're doing at least a few of these things.
 
+* Strategies to Establish Value
+    1. Connect the material to students' interests.
+    2. Provide authentic, real-world tasks.
+    3. Show relevance to sutdents' current academic lives.
+    4. Demonstrate the relevance of higher-level skills to students' future professional lives.
+    5. Identify and reward what you value. 
+    6. Show your own passion and enthusiasm for the discipline.
+* Strategies to Build Positive Expectations
+	1. Ensure alignment of objectives, assessments, and instructional strategies.  
+	2. Identify an appropriate level of challenge.
+	3. Create assignments that provide an appropriate level of challenge. 
+	4. Provide early success opportunities.  
+	5. Articulate your expectations. 
+	6. Provide rubrics.
+	7. Provide targeted feedback. 
+	8. Be fair. 
+	9. Educate students about the ways we explain success and failure.  
+	10. Describe effective study strategies.
+* Strategies for Self-Efficacy
+	1. Provide flexibility and control.  
```

Comment:
Not sure I understand this one without going back to the book. Can it be phrased more clearly?